libs/clients/health-directorate/src/lib/healthDirectorateClient.types.ts

```diff
@@ -0,0 +1,25 @@
+export interface HealthcareLicense {
```

Comment:
Are these all custom types? Or could they be imported from the generated client?